templates/careers/all.html

```diff
@@ -7,12 +7,15 @@
 {% block meta_description %}Canonical publishes Ubuntu, provides commercial services and solutions for Ubuntu, and works with hardware manufacturers, software vendors and public.{% endblock %}
 
 {% block hero %}
-  <section class="p-strip--image is-dark is-deep u-no-padding--top u-no-padding--bottom u-extra-space">
-    <div class="p-strip--suru-background">
-      <div class="row">
-        <div class="col-6">
-          <h1 class="u-no-margin--bottom">All roles</h1>
-        </div>
+  <section class="p-strip--light is-deep">
+    <div class="row">
+      <div class="col-3">
+        <h1 class="u-no-margin--bottom p-heading--2">Open roles</h1>
```

Comment:
"Opens roles" and the search bar are aligned in the same line..maybe because of `u-no-margin--bottom"? 
<img width="953" alt="image" src="https://user-images.githubusercontent.com/57550290/194058649-7bf62f00-e1da-4148-a2e9-2b127b6bdb51.png">

Feel free to disregard this too if you got design +1 already! :)

Reply:
It did get +1, but you're right that it looks weird. Removing the `u-no-margin-bottom` class did nothing so I've added some padding and hopefully it's looking better